department-of-veterans-affairs/abd-vro
# Feature - Update Classifier Mappings for New conditions

## What was the problem?

This PR is part 2 of 2 parts in order to update the lookup table mappings.

This PR includes updates to the ConditionDropdownLookupTable which is used for mapping autosuggestions and free text to classifications.
This PR also includes changes to use a new .csv file for capturing dropdown coverage in the datadog logs.
 - this was moved to a separate csv file in order to improve quality of life for folks doing the research for these mappings

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/459

## How does this fix it?[^1]

csv's for lookup tables are now split into three directories
 - `domain-cc/cc-app/src/python_src/util/data/condition_dropdown_coverage`
   - this is what is used to determine what is logged as `is_in_dropdown` in the datadog logs
 - `domain-cc/cc-app/src/python_src/util/data/condition_dropdown_lookup_table`
   - this is what maps autosuggestion and free text to classifications
 - `domain-cc/cc-app/src/python_src/util/data/dc_lookup_table`
   - this remains the same - diagnostic codes are mapped to classifications


## How to test this PR
 - `cd domain-cc/cc-app`
 - `pytest`
 - Note lookup table size has increased. There are 50 new mappings


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
